lib/pure/net.nim

```diff
@@ -1425,21 +1425,33 @@ proc recvFrom*(socket: Socket, data: var string, length: int,
   ## so when ``socket`` is buffered the non-buffered implementation will be
   ## used. Therefore if ``socket`` contains something in its buffer this
   ## function will make no effort to return it.
+  template recvFrom(domain: Domain) =
```

Comment:
Nit: rename this to something that doesn't match the outer proc. This is bound to get confusing.

Reply:
Renamed to adaptRecvFromToDomain().